packages/openneuro-app/src/scripts/datalad/dataset/dataset-query-fragments.js

```diff
@@ -97,3 +97,67 @@ export const SNAPSHOT_ISSUES = gql`
     }
   }
 `
+
+export const DATASET_METADATA = gql`
+  fragment DatasetMetadata on Dataset {
+    id
+    metadata {
+      context {
+        datasetId
+        datasetUrl
+        datasetName
+        firstSnapshotCreatedAt
+        latestSnapshotCreatedAt
+        dxStatus
+        tasksCompleted
+        trialCount
+        studyDesign
+        studyDomain
+        studyLongitudinal
+        dataProcessed
+        species
+        associatedPaperDOI {
+          id
+          type
+        }
+        openneuroPaperDOI {
+          id
+          type
+        }
+        seniorAuthor
+        adminUsers {
+          type
+          container
+        }
+        ages {
+          type
+          container
+        }
+        modalities {
+          type
+          container
+        }
+      }
```

Comment:
What's the reason for duplicating most of these fields in two formats via the context field?

Reply:
As far as I can tell, the context is the most standard way to define terms in JSON-LD documents. We could convert it to the more literal syntax where IRIs are the keys, but I think that sequestering them in the context makes for easier data management. I'm not sure that we need the context on the front-end at the moment, but I thought it would be useful in case we wanted to make the metadata downloadable in its JSON-LD format.